sap/stackstate_checks/sap/proxy.py

```diff
@@ -12,10 +12,23 @@ def __init__(self, url, user, password, verify=False, cert=None, keyfile=None):
         session = Session()
         if cert:
             session.verify = verify
+            if not verify:
+                # since `trust_env` is by default True and overrides `verify` flag with CURL_CA_BUNDLE certificate path
+                # set by Agent runtime for verification, making this flag `False` doesn't override and doesn't verify
+                # the certificate.
+                session.trust_env = False
             session.cert = (cert, keyfile)
         session.auth = HTTPBasicAuth(user, password)
         wsdl_url = "{0}/?wsdl".format(url)
         self.client = Client(wsdl_url, transport=Transport(session=session, timeout=10))
+        sap_type = wsdl_url.split("/")[-2]
+        address = "/".join(wsdl_url.split("/")[:-2])
+        # ServiceProxy for same host location from config as the host location can be different in WSDL response
```

Comment:
please provided both examples urls, and log the urls as well

Reply:
provided an example for URLs but logging.